spec/requests/api/pages_spec.rb

```diff
@@ -0,0 +1,116 @@
+require_relative 'shared_language_pages.rb'
+require 'rails_helper'
+
+describe "api/pages" do
+
+  def json
+    JSON.parse(response.body)
+  end
+
+
+  before :each do
```

Comment:
Without `each`, some of the specs sometimes fail for me because of time stamps being incomparable.